packages/redux/src/users/addresses/actionTypes.ts

```diff
@@ -94,6 +94,38 @@ export const REMOVE_USER_DEFAULT_CONTACT_ADDRESS_REQUEST =
 export const REMOVE_USER_DEFAULT_CONTACT_ADDRESS_SUCCESS =
   '@farfetch/blackout-redux/REMOVE_USER_DEFAULT_CONTACT_ADDRESS_SUCCESS';
 
+/**
+ * Action type dispatched when delete default contact address request fails.
```

Comment:
Update these descriptions please:

```suggestion
 * Action type dispatched when delete default billing address request fails.
```